noaa/challenges/default.json

```diff
@@ -342,8 +342,8 @@
       ]
     },
     {
-      "name": "sub-bucket-aggs",
-      "description": "Checks the performance of bucket aggregations under bucket aggregations",
```

Comment:
We don't run these in the nightlies so the name change is probably not a big deal.